system/lib/sbrk.c

```diff
@@ -41,7 +58,6 @@ void *sbrk(intptr_t increment) {
   intptr_t expected;
   while (1) {
 #endif // __EMSCRIPTEN_PTHREADS__
-
     intptr_t* sbrk_ptr = emscripten_get_sbrk_ptr();
```

Comment:
You can also drop this line and just `&sbrk_val` everywhere in this function instead?